pkg/test/framework/components/istio/kube.go

```diff
@@ -116,17 +116,20 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		return nil, err
 	}
 
-	// Wait for Galley & the validation webhook to come online before applying Istio configurations.
-	if _, _, err = env.WaitUntilServiceEndpointsAreReady(cfg.SystemNamespace, "istio-galley"); err != nil {
-		err = fmt.Errorf("error waiting %s/istio-galley service endpoints: %v", cfg.SystemNamespace, err)
-		scopes.CI.Info(err.Error())
-		return nil, err
-	}
+	if !cfg.DoNotWaitForValidationWebhook {
```

Comment:
nit: double negation, suggest structure as a block with an early exit.

Reply:
Renamed to Skip. This needs to be false by default.